Add support for Elusive and its modifiers, unify Wither and Withered

Quotae opened the pull request:
Added support for Elusive. The option only shows up if the character has something that checks for it, including any gems, the Assassin node, and Badge of the Brotherhood, everything but the Ascendant Assassin node. Effect of Elusive on passives, items, and gems scales the buff accordingly. It's a simple toggle, no support for the scaling effect of the buff because I couldn't figure out how to do it, but the tooltip mentions that it decays over time.

![image](https://user-images.githubusercontent.com/39030429/65732653-7c7d2700-e090-11e9-8fe6-c4349f3bf0d9.png)

Additionally, the Wither spell and other sources of Withered like Withering Touch now apply the same debuff, instead of being two separate debuffs that stack. The input box for WIthered Stacks is now in the Effective DPS column. The box now also shows up when Withering Step is equipped.

![image](https://user-images.githubusercontent.com/39030429/65733183-04fcc700-e093-11e9-9cd4-e2ffba8b0644.png)

Legrems commented:
You can already applied Withered to the enemy. Just add the skill wither, and make it 15 stacks :)

Legrems commented:
And by the way, you realy need to have usefull commit name ^^'

Quotae commented:
It's not just to apply Withered to the enemy, it's so that when you're using Withering Touch and the Wither spell, you can't apply both the Wither spell's 15 stacks AND the 15 Withering Touch stacks. Both the Wither spell and Withering Touch apply Withered, so they shouldn't stack separately, because they're the same debuff with a limit of 15 stacks. Also, Withering Step can supply Withered Stacks, so it doesn't make sense to have Withered Stacks under a skill-specific option, it should be in a general location.

And I left comments on all those commits, I just didn't realize I could change the name of them, lol. Got it this time though. Copy/paste changelog from the comment of that most recent commit:

- Added a display for current Elusive effect in the Calcs tab, under Movement Speed in Other Defences, which only shows up while Elusive.
![image](https://user-images.githubusercontent.com/39030429/66099173-64634700-e56b-11e9-8e45-e7125655801e.png)

- Much better support for Nightblade Support. Now gives Crit Multi and Base Crit Chance to attacks while using Claws or Daggers, which it didn't do at all until now.
- Tweaked how I implemented Elusive Effect on gems. Now it's implemented specifically for Claws/Daggers on Nightblade, and is added as a global Skill Mod for Withering Step and any future gems which give the stat.
- Are you Elusive? button works and shows up more consistently.

Hopefully that's thorough enough for you!
